frontend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# pull official base image
+FROM node:16.13.2-alpine
+
```

Comment:
```suggestion
NODE_ENV=production
```

So we build production things and don't install dev dependencies.

Reply:
Done 👍